src/Check/DirReadable.php

```diff
@@ -23,25 +19,15 @@
  */
 class DirReadable extends AbstractCheck implements CheckInterface
 {
-    /** @var array|Traversable */
+    /** @var array<string> */
     protected $dir;
 
     /**
-     * @param  string|array|Traversable $path Path name or an array of paths
+     * @param  string|array<string> $path Path name or an array of paths
      * @throws InvalidArgumentException
      */
-    public function __construct($path)
+    public function __construct(string|array $path)
```

Comment:
```suggestion
    public function __construct(string|iterable$path)
```

Traversable was allowed previously.